uber/site_sections/accounts.py

```diff
@@ -70,7 +70,7 @@ def login(self, session, message='', original_location=None, **params):
 
         if 'email' in params:
             try:
-                account = session.get_account_by_email(params['email'])
+                account = session.get_account_by_email(normalize_email(params['email']))
```

Comment:
Agree. The only thing to consider is that we'd like to be able to express the normalization in SQL. So we may end up with a more aggressive normalization for queries (while maintaining the original email as provided by the attendee).